.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+
+common: &common
+  working_directory: ~/raster-tester
+  steps:
+    - checkout
+    - run: sudo apt-get -y --no-install-recommends --force-yes install gdal-bin libgdal-dev
+    - run: sudo pip install virtualenv
+    - run: virtualenv venv
+    - run:
+        name: install dependencies
+        command: pip install --user tox codecov
+    - run:
+        name: run tox
+        command: ~/.local/bin/tox
+    - run:
+        name: upload coverage report
+        command: |
+          if [[ "$UPLOAD_COVERAGE" == 1 ]]; then
```

Comment:
`UPLOAD_COVERAGE` variable is set so we don't create coverage for any env (py2 and py3)